pkgs/by-name/np/npiet/package.nix

```diff
@@ -40,6 +47,7 @@ stdenv.mkDerivation rec {
     homepage = "https://www.bertnase.de/npiet/";
     license = licenses.gpl2Only;
     platforms = platforms.unix;
+    mainProgram = "npiet";
     maintainers = with maintainers; [ Luflosi ];
```

Comment:
One more thing you can do in the modernize commit is to remove the `with lib;` usage, and use `lib.licences`, `lib.platforms` and `with lib.maintainers;`.

Reply:
In my opinion this use of `with lib;` is perfectly fine since the scope is small and there is no confusion where the variables come from.
Is it documented anywhere that `with lib;` should not be used for `meta`?

Reply:
> In my opinion this use of `with lib;` is perfectly fine since the scope is small and there is no confusion where the variables come from. Is it documented anywhere that `with lib;` should not be used for `meta`?

Yea this is slightly debatable, See examples:

- https://github.com/NixOS/nixpkgs/pull/293767
- https://github.com/NixOS/nixpkgs/issues/208242
- https://github.com/NixOS/nixpkgs/issues/292468

Feel free to leave this as is, or if you decide to not change your opinion, cc me and I'll merge it as is.

Reply:
I'm still very much on the fence about the use of `with lib;` in `meta` but I have decided to apply your suggestion.